index.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Tangerine">
+          <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
+          <link href="https://fonts.googleapis.com/icon?family=Material+Symbols"
+          rel="stylesheet">
+  </head>
+  <body>
+    <section class="whole-main">
```

Comment:
- [ ] [optional] Kindly restore your header and the first section to your code. Your portfolio is not professional without the two sections. 👍